src/Component/Board/index.spec.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { createStore } from 'redux'
+import { Provider } from 'react-redux'
+import Board from './index'
+import State from '../../Redux/State'
+import TestRenderer from 'react-test-renderer'
+
+describe('Component :: Board :: index', () => {
+
+  it('checks the initial state and verify if the board is empty', () => {
+    const store = createStore(State, State(), undefined)
+    const inspector = TestRenderer.create(createElement(store, Board))
+
+    inspector.root.findByProps({ className: 'myBoard' })
```

Comment:
At this stage it's fine to just check if there is a component with a specific prop because the feature it's in an early stage.
When the feature will be finalized it would be better to assert via snapshots instead of just checking if there is a component with a specific prop.